### ISSUE TITLE
gem not working on github.io

### ISSUE THREAD

**Issue by riccardoklinger** [DavidJVitale/jekyll-leaflet#9]
I was adding the gem to my github.io page. There is a build error with this gem:
The tag `leaflet_map` on line 19 in `index.html` is not a recognized Liquid tag 
I do have:

```
group :jekyll_plugins do
  gem 'jekyll-paginate'
  gem 'jekyll-sitemap'
  gem 'jekyll-leaflet'
end
```
in my GemFile

and:
```
plugins:
  - jekyll-paginate
  - jekyll-sitemap
  - jekyll-leaflet
```
in my _config.yml

**Comment by DavidJVitale** [comment]
I do not believe GitHub pages supports third party gems/plugins (like Jekyll-leaflet). So I’m not sure you can use it with GitHub pages, unless you build yourself and push up manually. I’m not too familiar with GitHub pages so maybe someone else can chime in
